test/e2e/buildx_inspect_test.go

```diff
@@ -0,0 +1,209 @@
+//go:build linux || freebsd
+
+package integration
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman buildx inspect", func() {
+	It("podman-remote buildx inspect", func() {
+		if !registry.IsRemote() {
+			Skip("This test is only applicable for remote podman")
+		}
+
+		session := podmanTest.PodmanExitCleanly("buildx", "inspect")
+		output := session.OutputToString()
+
+		infoSession := podmanTest.PodmanExitCleanly("info", "--format", "{{.Host.OS}}/{{.Host.Arch}}")
+		expectedNativePlatform := strings.TrimSpace(infoSession.OutputToString())
+
+		Expect(output).To(MatchRegexp(`Platforms:\s+.*\b` + regexp.QuoteMeta(expectedNativePlatform) + `\b`))
+	})
+
+	It("podman-remote buildx inspect --bootstrap", func() {
+		if !registry.IsRemote() {
+			Skip("This test is only applicable for remote podman")
+		}
+
+		session := podmanTest.PodmanExitCleanly("buildx", "inspect", "--bootstrap")
+		output := session.OutputToString()
+
+		infoSession := podmanTest.PodmanExitCleanly("info", "--format", "{{.Host.OS}}/{{.Host.Arch}}")
+		expectedNativePlatform := strings.TrimSpace(infoSession.OutputToString())
+
+		Expect(output).To(MatchRegexp(`Platforms:\s+.*\b` + regexp.QuoteMeta(expectedNativePlatform) + `\b`))
+	})
+
+	It("podman buildx inspect", func() {
+		if registry.IsRemote() {
+			Skip("This test is only applicable for local podman")
+		}
+
+		session := podmanTest.PodmanExitCleanly("buildx", "inspect")
+		stdErr := session.ErrorToString()
+		Expect(stdErr).ToNot(MatchRegexp(`(?i)error starting|fail|fatal|panic`), "stderr should not contain critical error keywords, got: %s", stdErr)
```

Comment:
this is not checking anything ExitCleanly() verifies already that there is not stderr output so there is no reason to regex check any of its output.